lib/kvs/kvs_rparams.c

```diff
@@ -186,11 +186,11 @@ static const struct param_spec pspecs[] = {
         .ps_stringify = param_default_stringify,
         .ps_jsonify = param_default_jsonify,
         .ps_default_value = {
-            .as_uscalar = 16,
+            .as_uscalar = 19,
         },
         .ps_bounds = {
             .as_uscalar = {
-                .ps_min = 4,
+                .ps_min = 8,
```

Comment:
What necessitated the limit change here?

Reply:
After a 4G split you'd be left with 2 2G nodes, things could get real bad real fast, and I don't want to have to deal with customers who shoot themselves in the foot.  I am actually thinking 13 might be a better lower bound (cuz I like primes!), but we have time to evaluate.